src/lib/urn.ts

```diff
@@ -117,9 +126,18 @@ export function isThirdParty(urn?: string) {
   return decodedURN.type === URNType.COLLECTIONS_THIRDPARTY
 }
 
+export function extractEntityId(urn: URN): string {
+  const decodedURN = decodeURN(urn)
+  if (decodedURN.type !== URNType.ENTITY) {
+    throw new Error('URN is not a third party URN')
+  }
+
+  return decodedURN.entityId ?? ''
+}
+
 export function decodeURN(urn: URN): DecodedURN {
   const urnRegExp = new RegExp(
-    `${baseMatcher}:${protocolMatcher}:${typeMatcher}:(?<suffix>${baseAvatarsSuffixMatcher}|${collectionsSuffixMatcher}|${thirdPartySuffixMatcher})`
+    `${baseMatcher}:(${protocolMatcher}:)?${typeMatcher}:(?<suffix>${baseAvatarsSuffixMatcher}|${collectionsSuffixMatcher}|${thirdPartySuffixMatcher}|${entitySuffixMatcher})`
```

Comment:
Updated regex to match urn that contains the format: `urn:decentraland:entity:<EntityId>?baseUrl=<ContentServerURL>`